central/cluster/datastore/telemetry.go

```diff
@@ -2,7 +2,10 @@ package datastore
 
 import (
 	"context"
+	"reflect"
+	"time"
 
+	"github.com/hashicorp/golang-lru/v2/expirable"
```

Comment:
We already have https://github.com/stackrox/stackrox/blob/master/pkg/expiringcache/cache.go
We should decide to use one unless there is a reason to have 2 :)